tests/src/Unit/RedirectRequestSubscriberTest.php

```diff
@@ -252,6 +196,9 @@ protected function getRedirectStub($url, $status_code = 301) {
     $redirect->expects($this->any())
       ->method('id')
       ->willReturn(1);
+    $redirect->expects($this->once())
```

Comment:
Would be nice to have unit test coverage for the cache tag header but we're not validating the returned redirect at all right now, which is sad, but not the problem of this issue to solve. Can we mock this properly though, and return 'redirect:1', so that it matches the real implementation?